client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/purchase-section/choose-credit-card-button.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
I noticed this file is very similar to https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/purchase-section/add-credit-card-button.js.  

First thought was maybe we can create a single component instead of 2 components, and then pass label/description/url/functions as props instead of duplicating the same functionalities. I think same component also make sense because they are sharing quite a bit of similar functionality and layout (button design).

Thinking about trade offs, the benefit is that it helps future maintenance since it reduces duplication. The con is that these two buttons do have differences so the parent component would need to handle callback functions, translation, URL and the refactoring the "Add credit card" button, seems to be out of scope and we won't meet our timeline. 

If we are creating more of these buttons or if these 2 components are always going to show up together, then maybe it is a good idea to make 1 component instead of 2. I don't think either of these is the case for us here, so I think we are okay with 2 separate components even though they share some of the same codes.

Reply:
Thanks @harriswong. I created https://github.com/Automattic/woocommerce-services/issues/1799 for this.